src/js/components/Meter/stories/Labelled.js

```diff
@@ -4,26 +4,30 @@ import { storiesOf } from '@storybook/react';
 import { Grommet, Box, Meter, Stack, Text } from 'grommet';
 import { grommet } from 'grommet/themes';
 
-const LabelledMeter = () => (
-  <Grommet theme={grommet}>
-    <Box align="center" pad="large">
-      <Stack anchor="center">
-        <Meter
-          type="circle"
-          background="light-2"
-          values={[{ value: 30 }]}
-          size="xsmall"
-          thickness="small"
-        />
-        <Box direction="row" align="center" pad={{ bottom: 'xsmall' }}>
-          <Text size="xlarge" weight="bold">
-            30
-          </Text>
-          <Text size="small">%</Text>
-        </Box>
-      </Stack>
-    </Box>
-  </Grommet>
-);
+function LabelledMeter() {
+  const meterValue = 30;
```

Comment:
I believe you can declare this variable outside this component.

Reply:
I could, but it's such a minor point and there are pros and cons both ways.

The only pro I can think of with it being outside of that component is it will only ever be declared and initialised once, so you are saving that initialisation whenever that function gets called. If we were writing a kernel here I may agree :)

The pro of leaving it inside, is that in general, variables should have the smallest scope they need to do their job. The meterValue variable is only relevant to that component so it should be encapsulated by it

Reply:
agreed lets keep it this way. 
1) no need to be reviewing such minor details especially in stories. 
2) the user should be able to get the code from the story in an easy way. putting this elsewhere will reduce readibility in storybook